src/calculator/validation.ts

```diff
@@ -20,6 +20,9 @@ export function validate(method: string) : any {
       check('weightMisspelling', 'weightMisspelling.REQUIRED').exists(),
       check('weightMisspelling', 'weightMisspelling.NUMBER').isFloat(),
       check('weightMisspelling', 'weightMisspelling.NOT_IN_RANGE').isFloat({min : 0, max : 100}),
+      check('WEIGHT_TEXT_CRED_SUM_NOT_1', 'customValidation.WEIGHT_TEXT_CRED_NOT_EQUALS_TO_1')
+        .custom((val: string, obj: any) => 
+          parseFloat(obj.req.query.weightSpam) + parseFloat(obj.req.query.weightBadWords) + parseFloat(obj.req.query.weightMisspelling) == 1),
```

Comment:
You should compare using === instead of ==. Also, since this parameters are footing number i think that we should check the validation using an epsilon.